From df3ca5378f5937afb773ff0207ae3d22fb77129f Mon Sep 17 00:00:00 2001
From: "Kan, Chris" <chris.kan@intel.com>
Date: Tue, 2 Sep 2014 10:29:46 +0200
Subject: [PATCH 091/117] ov5693: correct max coarse integration time for 3A
 enabling.

Change max coarse integration time to be in sync with
3A libs.

Issue: GMINL-143
Change-Id: Id5b44a50b354b83be7526de7bf8cdb051a07a802
Signed-off-by: Mikael Persson <mikael.persson@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/ov5693.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov5693.h b/drivers/external_drivers/camera/drivers/media/i2c/ov5693.h
index cbcfe82..c71b9d6 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov5693.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov5693.h
@@ -80,7 +80,7 @@
 #define OV5693_FINE_INTG_TIME_MIN 0
 #define OV5693_FINE_INTG_TIME_MAX_MARGIN 0
 #define OV5693_COARSE_INTG_TIME_MIN 1
-#define OV5693_COARSE_INTG_TIME_MAX_MARGIN (0xffff - 6)
+#define OV5693_COARSE_INTG_TIME_MAX_MARGIN 6
 
 #define OV5693_BIN_FACTOR_MAX 4
 /*
-- 
1.9.3

